### PR TITLE
Hello microverse first project

### PR DESCRIPTION
**Pull request by Surafels**
hello microverse project

- Added linter

- Added hello microverse!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,2 @@
-# .gitignore
-node_modules/
-test.md
\ No newline at end of file
+node_modules
+test.
```
Comment (Olamarx) [inline .gitignore]: - [ ] I can see that the test.md file still shows in the repo, just as node_modules is not showing, kindly ensure test.md is also not showing. I can see that you only wrote `test.` which is not the complete name of the file and as a result, is not attached to it, kindly complete the name.

File: README.md [hunk 2]
```diff
@@ -27,8 +26,8 @@
 - [🙏 Acknowledgements](#acknowledgements)
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
-# 📖 [project_name] <a name="Hello_Microverse_project"></a>
-**[Hello_Microverse_project]** is describing greeting for Microverese 
+# 📖 [project_name] <a name="Hello Microverse project"></a>
+**[Hello Microverse project]** is describing greeting for Microverse
```
Comment (Olamarx) [inline README.md]: - [ ] I will encourage you to add the actual project name to make your repo professional.

image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Surafels, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- All linters passed ✔️
- Readme file headers ✔️
- Commit messages meaningful ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by skyv26 — CHANGES_REQUESTED** [review]
## Hi @Surafels, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## STATUS: REQUIRE CHANGES 🔴 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 

## Required Changes ♻️
- [ ] Well done 👏🏻 , you did an awesome job by implementing the changes as per the previous code reviewer's request 👍🏻 but please always adhere to the `project guidelines`. As per project requirement, you need to create the `test. md` file and put the path of the file into the `.gitignore`. It will help you to understand how the `.gitignore` help us to ignore the unwanted file in your repo while staging.

- Please look at the below snippet in order to fulfil the project requirements.
Screenshot from 2023-04-25 17-01-27

### Steps for solving the issue

1. Delete the `test.md` file and make the commit.
2. Make sure that the `test.md` file should be ignored by `,gitignore` and make the commit.
3. Add back the `test.md` file and now after this step if you try to make commit the test.md file will be ignored.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by skyv26** [comment]
@Surafels Well done! You ignored the `test.md` file by following the steps.

**Review by Olamarx — APPROVED** [review]
Hi @Surafels, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- All required changes fixed ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**